app/src/main/java/com/karumi/ui/domain/model/SuperHero.kt

```diff
@@ -0,0 +1,6 @@
+package com.karumi.ui.domain.model
+
+data class SuperHero(val name: String,
+    val photo: String,
```

Comment:
If I remember well. The Marvel API has the picture field as optional.

Reply:
the problem is if this is optional picasso crash, what do you prefer? check in ui or check in the respository layer?

Reply:
If you change the picture to optional you'll have to check if the picture is defined before using it in picasso.